README.md

```diff
@@ -938,6 +946,73 @@ const info = await nona.rpc('account_info', {
 });
 ```
 
+## Name Service
+
+You can access to the `nameService` object with the following code:
+
+\```typescript
+const nameService = nona.nameService;
+\```
```

Comment:
Sorry in my last suggestion I had to put backslash to avoid closing my github suggestion but there was applied to the doc. Can you removed them ?

Reply:
Good catch. Thanks for all the feedback!